target/postfix/main.cf

```diff
@@ -56,6 +56,9 @@ smtpd_client_restrictions = permit_mynetworks, permit_sasl_authenticated, reject
 smtpd_sender_restrictions = $dms_smtpd_sender_restrictions
 smtpd_discard_ehlo_keywords = silent-discard, dsn
 disable_vrfy_command = yes
+smtpd_data_restrictions = reject_unauth_pipelining
+# TODO enable when possible, see https://github.com/docker-mailserver/docker-mailserver/issues/3719#issuecomment-1868287208
+#smtpd_forbid_bare_newline = yes
```

Comment:
"When possible" could better clarify that the blocker is waiting on Debian to provide an updated package of Postfix (_3.8.4, 3.7.9, 3.6.13 and 3.5.23_). For Bookworm that'd be `3.7.9`.

The comment also only refers to enabling the parameter, despite obsoleting the [short-term workarounds](https://www.postfix.org/smtp-smuggling.html#short).

---

The [longterm-fix](https://www.postfix.org/smtp-smuggling.html#long) also references an optional 2nd parameter to configure:

```
smtpd_forbid_bare_newline = yes
smtpd_forbid_bare_newline_exclusions = $mynetworks
```

We shouldn't configure it by default in the image though (_since Docker with IPv6 hosts but IPv4 only containers with default `user-proxy: true` daemon config_) can be part of the trusted docker subnet (_depending on our `PERMIT_DOCKER` ENV_). Those that need it (_like our test-suite_) could enable it, and we can document that in our docs somewhere.